src/qililab/instruments/qblox/qblox_module.py

```diff
@@ -412,10 +419,16 @@ def _set_hardware_modulation(self, value: float | str | bool, sequencer_id: int)
             ValueError: when value type is not bool
         """
         self._get_sequencer_by_id(id=sequencer_id).hardware_modulation = bool(value)
-        self.device.sequencers[sequencer_id].mod_en_awg(bool(value))
+
+        if Instrument.is_device_initialized(self):
+            self.device.sequencers[sequencer_id].mod_en_awg(bool(value))
 
     @Instrument.CheckParameterValueFloatOrInt
-    def _set_frequency(self, value: float | str | bool, sequencer_id: int):
+    def _set_frequency(
+        self,
+        value: float | str | bool,
+        sequencer_id: int,
+    ):
```

Comment:
Do you have the correct `line-length` setup in your VSCode ([look at the last setting of the JSON](https://www.notion.so/qilimanjaro/Code-guidelines-199b362b39af4261a2eaa512051145bb?pvs=4#fba4f5ca731d4b019e508d9134eb1c07))?

If you do, removing the comma of the last argument and saving should move the code back to a single line. 🙏

Reply:
This way the PR will have much less line changes!

Reply:
Oh! I didn't know a coma made the auto-formater put it in vertical!

I'll change it, sure!